src/main/java/org/springframework/samples/petclinic/system/PetClinicToggles.java

```diff
@@ -13,14 +13,17 @@
 
     public static Toggle toggleFindOwnerByLastName = new Toggle(Toggle.ALWAYS_ON_WHEN_ENABLED);
     public static Toggle toggleListOfOwners = new Toggle(0.5f);
+    public static Toggle toggleFindOwnerByLocation = new Toggle(0.65f);
     public static Toggle toggleFindOwnerByFirstName = new Toggle(0.5f);
 
     // TODO remove
-   
+    public static Collection<Boolean> toggles = new ArrayList<>(Arrays.asList(toggleFindOwnerByLastName.isOn(), toggleFindOwnerByLocation.isOn(), toggleListOfOwners.isOn()));
+
 
     public static Collection<Boolean> getToggleValues() {
         return new ArrayList<>(Arrays.asList(
             toggleFindOwnerByLastName.isOn(),
+            toggleFindOwnerByLocation.isOn(),
```

Comment:
Add you toggle to bottom of array and use toggles[3] in frontend to refer to it instead